src/features/student/components/toggles/TraceAuthenticDeclarationToggle/TraceAuthenticDeclarationToggle.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import { AvToggle, type AvToggleProps } from '@avenirs-esr/avenirs-dsav'
+import { useAttrs } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+interface TraceAuthenticDeclarationToggleProps extends Omit<AvToggleProps, 'description' | 'activeText' | 'inactiveText'> {
+  description?: string
+  activeText?: string
+  inactiveText?: string
+}
```

Comment:
```suggestion
interface TraceAuthenticDeclarationToggleProps extends Omit<AvToggleProps, 'description' | 'activeText' | 'inactiveText'> 
```
A priori tu n'utilises jamais ces propriétés dans ton code

Reply:
@p-beger c'est un extend de type pour les rendre optional mais pour activeText  et inactiveText t'as raison ils sont deja optionnel

Reply:
@elrhourha je sais mais là a priori tu n'as jamais eu besoin de passer des custom activeText ou inactiveText donc ça sert à rien d'exposer ces propriétés